src/sentry/integrations/perforce/repository.py

```diff
@@ -48,51 +88,133 @@ def build_repository_config(
         Returns:
             Repository configuration
         """
+        depot_path = data["identifier"]
```

Comment:
@cathteng it would be great to have a more structured data type here:
https://github.com/getsentry/sentry/blob/a93aafe6f45205a09cbbea30a7aee04a5bc188e6/src/sentry/integrations/github/repository.py#L55